tests/spec/fs.rmdir.spec.js

```diff
@@ -97,4 +97,27 @@ describe('fs.rmdir', function() {
       });
     });
   });
+
+  it('(promise) should be a function', function() {
```

Comment:
Just to nitpick here, you can use () => {...} instead of function () {...}. It's a shorter syntax.

Reply:
@YuechengWu other PR comments on this syntax suggestion seem to recommend maintaining consistent syntax throughout the file; because the other (non-promise) cases are not using the arrow syntax, it might be best to leave it as is.